src/routes/activityReports/handlers.js

```diff
@@ -722,6 +722,14 @@ export async function getActivityRecipients(req, res) {
   res.json(activityRecipients);
 }
 
+export async function getActivityRecipientsForExistingReport(req, res) {
```

Comment:
Do we need to check permissions here?

Reply:
I'm not actually sure. What do you think? What we're potentially exposing (to already-authenticated users) is which grant recipients were previously associated with a given AR. My initial thought is that this isn't sensitive - but maybe I"m wrong

Reply:
I suppose without context that info doesn't feel particularly sensitive. The only counterargument is that all the other report-related data is confined to a region. I'm ok with site-access level permissions here, personally. Although, I am not the decider.

Reply:
I ended up adding AR read permissions to this handler: 440a55a